src/contrib/cluster/Akka.Cluster.Tools.Tests/PublishSubscribe/DistributedPubSubMediatorSpec.cs

```diff
@@ -54,5 +56,58 @@ public void DistributedPubSubMediator_can_be_activated_in_child_actor()
                 a.ShouldNotBe(ActorRefs.Nobody);
             });
         }
+
+        [Fact]
+        public void DistributedPubSubMediator_should_get_dead_letter()
+        {
+            EventFilter.Exception<NullReferenceException>().Expect(0, () =>
+            {
+                var actor = Sys.ActorOf((dsl, context) =>
+                {
+                    IActorRef mediator = null;
+                    ICancelable cancelable = null;
+                    dsl.OnPreStart = actorContext =>
+                    {
+                        Sys.EventStream.Subscribe(context.Self, typeof(DeadLetter));
+                        mediator = DistributedPubSub.Get(actorContext.System).Mediator;
```

Comment:
move the "arrange" steps to outside of the actor, to make things more comprehensible